Task3_Position_the_Cubes/index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <style>
+        *{
+            box-sizing: border-box;
+            margin: 0px;
+            padding: 0px;
+        }
+        .c1{
+         border: 2px solid white;
+         background-color: red;
+         height: 100px;
+         width: 100px;
+         position: relative;
+         top: 0px;
+         left: 0px;
+        } 
+        .c2{
+         border: 2px solid white;
+         background-color: green;
+         height: 100px;
+         width: 100px;
+         /* position: relative; */
+         top: 0px;
+         left: 0px;
+         position: absolute;
+         top: 100px;
+         left: 100px;
+         /* bottom: 0px; */
+        } 
+        .c3{
+         border: 2px solid white;
+         background-color: purple;
+         height: 100px;
+         width: 100px;
+         /* position: relative; */
+         position: absolute;
+         top: 200px;
+         left: 200px;
+        } 
+        .container{
+         border: 5px solid white;
+         height: 300px;
+         background-color: yellow;
+         height: 300px;

```

Comment:
The 'height' property is duplicated in the CSS block for '.container'. You may want to remove one of them to avoid redundancy and potential styling conflicts.